osu.Server.Queues.ScoreStatisticsProcessor/Processors/PlayCountProcessor.cs

```diff
@@ -47,7 +47,7 @@ public void ApplyToUserStats(SoloScoreInfo score, UserStats userStats, MySqlConn
             const int beatmap_count = 12;
             const int over_time = 120;
 
-            int secondsForRecentScores = conn.QuerySingleOrDefault<int?>("SELECT UNIX_TIMESTAMP(NOW()) - UNIX_TIMESTAMP(created_at) FROM scores WHERE user_id = @user_id AND ruleset_id = @ruleset_id ORDER BY id DESC LIMIT 1 OFFSET @beatmap_count", new
+            int secondsForRecentScores = conn.QuerySingleOrDefault<int?>("SELECT UNIX_TIMESTAMP(NOW()) - UNIX_TIMESTAMP(ended_at) FROM scores WHERE user_id = @user_id AND ruleset_id = @ruleset_id ORDER BY id DESC LIMIT 1 OFFSET @beatmap_count", new
```

Comment:
Is this intentionally changed? I'd expect

```suggestion
            int secondsForRecentScores = conn.QuerySingleOrDefault<int?>("SELECT UNIX_TIMESTAMP(NOW()) - UNIX_TIMESTAMP(started_at) FROM scores WHERE user_id = @user_id AND ruleset_id = @ruleset_id ORDER BY id DESC LIMIT 1 OFFSET @beatmap_count", new
```

Reply:
`created_at` used to be the database insert time, as far as i can tell. that no longer exists, and `ended_at` matches the time the user finished the play.

maybe it's been wrong until now? i'll have to understand what is being hoped for here